app/views/catalog/_document_list.html.erb

```diff
@@ -0,0 +1,12 @@
+<% # container for all documents in index list view -%>
+<div id="documents" class="documents-<%= document_index_view_type %>">
+  <% documents.in_groups_of(4, false) do |document_group| %>
+    <div class="row">
+      <% document_group.each do |document| %>
+        <div class="col-md-3">
+          <%= render document %>
+        </div>
+      <% end %>
+    </div>
+  <% end %>
+</div>
```

Comment:
@afred: Sort-of... except that when I tried just passing it as a local variable, that didn't work. For thinking about how to go forward, I think we should clarify whether for us it just makes the most sense to gut the original templates and keep our code as simple as possible... or if we want to preserve their structure as much as possible. I don't know.